python_modules/dagster/dagster/_core/definitions/schedule_definition.py

```diff
@@ -559,7 +559,7 @@ def with_updated_job(self, new_job: ExecutableDefinition) -> "ScheduleDefinition
             required_resource_keys=self._raw_required_resource_keys,
             run_config=None,  # run_config, tags, should_execute encapsulated in execution_fn
             run_config_fn=None,
-            tags=None,
+            tags=self.tags,
```

Comment:
Not related to this PR, but while you're here, i noticed that the docstring at line 546 uses the wrong param name